create_jenkins_job.py

```diff
@@ -252,6 +252,16 @@ def create_job(os_name, job_name, template_file, additional_dict):
             'ubuntu_distro': ubuntu_distro,
         })
 
+        ubuntu_distro = 'xenial'
+        # configure a nightly xenial packaging job
+        job_name = 'packaging_{0}_{1}'.format(ubuntu_distro, os_name)
+        create_job(os_name, job_name, 'packaging_job.xml.em', {
+            'cmake_build_type': 'RelWithDebInfo',
+            'test_bridge_default': 'true',
+            'time_trigger_spec': PERIODIC_JOB_SPEC,
+            'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
```

Comment:
FYI, this accidentally left out passing the ubuntu distro through, which was fixed up in https://github.com/ros2/ci/commit/dce075c47ccad6e5fc397699e56f6c9e9cca36ce